### PR TITLE
Restore linear colorscales

### PR DESCRIPTION
**Pull request by bmschmidt**
The d3 scales for colors were gone, affecting the ability to create legends, etc. This adds them back with appropriate types.

* Fixes NOM-1630

### CODE REVIEW

File: dev/svelte/ColorChange.svelte [hunk 1]
```diff
@@ -2,7 +2,7 @@
   export let scatterplot;
 
   let value = 'category10';
-  const schemes = ['okabe', 'category10', 'dark2', 'pastel2'];
+  const schemes = ['okabe', 'category10', 'dark2', 'pastel2', 'observable10'];
```
Comment (bmschmidt) [inline dev/svelte/ColorChange.svelte:5]: Adding one new colorscheme while we're in here.

File: package.json [hunk 2]
```diff
@@ -53,12 +53,12 @@
...
     "d3-random": "^3.0.1",
     "d3-scale": "^4.0.2",
-    "d3-scale-chromatic": "^3.0.0",
+    "d3-scale-chromatic": "^3.1.0",
     "d3-selection": "^3.0.0",
     "d3-timer": "^3.0.1",
     "d3-transition": "^3.0.1",
     "d3-zoom": "^3.0.0",
-    "deepscatter": "^3.0.0-next.6",
+    "deepscatter": "^3.0.0-next.23",
```
Comment (bmschmidt) [inline package.json:61]:
```suggestion
```

File: dev/svelte/ColorChange.svelte (hunk 1 above)
Comment (AndriyMulyar) [reply dev/svelte/ColorChange.svelte:5]: can we call it nomic10

**Review by AndriyMulyar — APPROVED** [review]

File: dev/svelte/ColorChange.svelte (hunk 1 above)
Comment (bmschmidt) [reply dev/svelte/ColorChange.svelte:5]: Nomic 10 will be even awesomer

**Pull request merged**